### ISSUE TITLE
Fetch may be blocked (on pause) if SubmitSapJob gets terminated prematurely

### ISSUE THREAD

**Issue by 3three** [3three/envielogistique#13]
FetchSapJob will automatically pause if it detects that there are any Rounds in the "Submitting" state, which is what happens during a SubmitSapJob.

SubmitSapJob has an Ensure block to ensure that whether it exists properly or not, the Round is moved out of the Submitting state, unblocking any FetchSapJob that may be there.

There is a risk, under very rare conditions, that the SubmitSapJob will terminate uncleanly and this ensure does not happen. Specifically, if the server is suddenly power cycled in the middle of SubmitSapJob. The consequence of this risk is that A) The SubmitSapJob never completes, and B) FetchSapJob is never unblocked.

So, in other words, this is very unlikely to occur, but the consequences are that the entire application ceases to work.

To unblock this problem, all Rounds in the "Submitting" state must be manually placed into another state (e.g., "Nouveau" or "Erreur").
